### DISCUSSION TITLE
Bagaimana cara (contoh) menggunakan fungsi pad untuk parameter start_val dan end_val ?

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#627] [Q&A] [answered]
Setelah dicobakan code pada fungsi pad , didapatkan  error sbb ;
<img width="649" alt="image" src="https://user-images.githubusercontent.com/123436963/231648765-83a13b17-41b8-436d-aba5-717ce64a27ea.png">
 Bagaimanakah cara memperbaiki cara penulisan parameter start_val dan end_val agar waktu yg dimulai pada setiap harinya adalah jam 00:00:00 dan berakhir di 23:00:00 ?

Terimakasih

**Comment by amaliasf** [comment, marked as answer]
Dear Pak @Micoalgo 

Terimakasih atas pertanyaanya Pak. Untuk parameter start_val dan end_val pada fungsi pad() seharusnya memiliki informasi datetime yang lengkap (tanggal dan waktu.)

Misal: `start_val = ymd_hms("2017-12-01 10:00:00 UTC")`

Pada code yang Bapak lampirkan nilai-nya masih belum sesuai maka mengeluarkan error.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Comment by Micoalgo** [comment]
Terimakasih jawabannya Mba Cut
